CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -44,6 +44,12 @@ public async Task<Result<AccountDto>> CreateAccountAsync(CreateAccountDto accoun
                     FirstName = accountModel.FirstName,
                     LastName = accountModel.LastName
                 };
+                string answerFromPasswordValidation = PasswordHelper.PasswordValidation(accountModel.Password);
+
+                if (answerFromPasswordValidation.Length!=0)
```

Comment:
I suggest to use **!string.IsNullOrEmpty(answerFromPasswordValidation)** instead of **answerFromPasswordValidation.Length!=0**

Reply:
Ok [update](f13709ef069680c412435ae752e7f7bd32dea95e)